lang/src/test/kotlin/org/partiql/lang/eval/CastTestBase.kt

```diff
@@ -214,6 +215,15 @@ abstract class CastTestBase : EvaluatorTestBase() {
         override fun toString() = "$castCase$additionalDescription"
     }
 
+    /**
+     * TODO fix tests — this is static final wasteland of util methods; not a test class.
+     *
+     * This is considered a test class by junit because TestBase uses the junit4 RunWith annotation.
+     *
+     * Consider replacing junit4 with junit5 because this is out of hand
+     * https://resources.jetbrains.com/storage/products/kotlinconf2018/slides/4_Best%20Practices%20for%20Unit%20Testing%20in%20Kotlin.pdf
+     */
+    @Ignore
```

Comment:
CastTestBase is marked as a test class (extends EvaluatorTestBase -> TestBase), but it doesn't actually contain a single test. It's also a companion object, so a static final class that just holds a bunch of static final methods to be used with testing ... yet it's not actually a source for tests. It's as if it was created to access the run/assert methods of TestBase, but be static so that ConfiguredCastArguments et. al. could access the `allConfiguredTestCases`. Either way, it's bad OOP. Without ignore, Gradle was unhappy. I can't explain why, but the ignore made Gradle relax.